File: 0001_0100/0082/0082.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    f.readline()
    s = [int(x) for x in (set(x for x in f.readline().split()) & set(x for x in f.readline().split()))]
    s.sort()
    q.write(" ".join(str(x) for x in s))

File: 0001_0100/0082/0082.go
package main

import (
	"fmt"
	"io/ioutil"
	"os"
	"strconv"
	"strings"
)

func main() {
	c, _ := ioutil.ReadFile("input.txt")
	q, _ := os.Create("output.txt")
	s := strings.Fields(string(c))
	var d [100001]byte
	n, _ := strconv.Atoi(s[0])
	a, b := s[2:2+n], s[2+n:]
	for _, v := range a {
		t, _ := strconv.Atoi(v)
		d[t] = 1
	}
	for _, v := range b {
		t, _ := strconv.Atoi(v)
		if d[t] == 1 {
			d[t] = 2
		}
	}
	for i, v := range d {
		if v == 2 {
			fmt.Fprint(q, i, " ")
		}
	}
}
